client/src/Components/Comment.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import calculateTimeElapsed from "../utils/getTime";
+import { FavoriteBorder, Favorite } from "@material-ui/icons";
+import authFetch from "../utils/authFetch";
+export default function Comment(props) {
+  const [likes, setLikes] = useState(false);
+  const [likeId, setId] = useState("");
+  useEffect(() => {
+    authFetch(`/commentsLike/${props.comment._id}`, null, props.history).then(
+      res => {
+        if (res) {
```

Comment:
This path is not RESTful.  There is a RESTful path in the likes-integration feature

Reply:
Sorry, Benjamin, I'll have to read your document about REST API naming conventions.